src/main/java/ua/bank/bankservice/service/impl/TransactionServiceImpl.java

```diff
@@ -0,0 +1,59 @@
+package ua.bank.bankservice.service.impl;
+
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.List;
+import lombok.AllArgsConstructor;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Sort;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import ua.bank.bankservice.exception.NotEnoughMoneyException;
+import ua.bank.bankservice.model.Account;
+import ua.bank.bankservice.model.Transaction;
+import ua.bank.bankservice.repository.TransactionRepository;
+import ua.bank.bankservice.service.AccountService;
+import ua.bank.bankservice.service.TransactionService;
+
+@Service
+@AllArgsConstructor
+public class TransactionServiceImpl implements TransactionService {
+    private final TransactionRepository transactionRepository;
+    private final AccountService accountService;
+
+    @Override
+    public List<Transaction> getAllByAccount(int page, int size, Account account) {
+        PageRequest date = PageRequest.of(page, size,
+                Sort.by("date").descending().and(Sort.by("id")));
+        return transactionRepository.getAllByAccountNumber(account.getAccountNumber(), date);
+    }
+
+    @Override
+    @Transactional
+    public void transfer(String fromAccount, String toAccount, int amount) {
```

Comment:
```suggestion
    public void transfer(Account fromAccount, Account toAccount, int amount) {
```
maybe let's move this to accountService?